modules/agent-install-sample-config-bonds-vlans.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_content-type: REFERENCE
```

Comment:
@mhanss can you please take a look ? (line 9 , there is a typo for VLAN) . Based on slack feedback added an example for bonds/vlans.

Reply:
@celebdor @bfournie I need a tech review on a priority basis for the example added for bonds/vlans. Please review this file.

Reply:
@Amrita42 lgtm